build.sbt

```diff
@@ -984,6 +984,7 @@ lazy val liteEngineKafkaRuntime: Project = (project in lite("kafka/runtime")).
     libraryDependencies ++= Seq(
       "commons-io" % "commons-io" % commonsIOV,
       "com.lightbend.akka.management" %% "akka-management" % akkaManagementV,
+      "com.typesafe.akka" %% "akka-slf4j" % akkaV,
```

Comment:
when I try to NussknackerApp I get `Caused by: java.lang.ClassNotFoundException: akka.event.slf4j.Slf4jLoggingFilter`